src/continuum_opacity/hydrogenic_bf_ff.jl

```diff
@@ -148,23 +148,23 @@ integral.
 
 # Arguments
 - `Z::Integer`: Z is the atomic number of the species (e.g. 1 for H I or 2 for He II)
-- `nmax_explicit_sum::Integer`: The highest energy level whose opacity contribution is included in
-   the explicit sum. The contributions from higher levels are included in the integral.
 - `nsdens_div_partition::Flt` is the total number density of the species divided by the species's
    partition function.
 - `ν::Flt`: frequency in Hz
 - `ρ::Flt`: mass density in g/cm³
 - `T::Flt`: temperature in K
 - `ion_energy::AbstractFloat`: the ionization energy from the ground state (in eV). This can be 
    estimated as Z²*Rydberg_H (Rydberg_H is the ionization energy of Hydrogen)
+- `nmax_explicit_sum::Integer`: The highest energy level whose opacity contribution is included in
+   the explicit sum. The contributions from higher levels are included in the integral.
 - `integrate_high_n::bool`: When this is `false`, bf opacity from higher energy states are not
    estimated at all. Default is `true`.
 
 # Notes
 This follows the approach described in section 5.1 of Kurucz (1970).
 """
-function hydrogenic_bf_opacity(Z::Integer, nmax_explicit_sum::Integer, nsdens_div_partition::Flt,
-                               ν::Flt, ρ::Flt, T::Flt, ion_energy::Flt,
+function hydrogenic_bf_opacity(Z::Integer, nsdens_div_partition::Flt, ν::Flt, ρ::Flt, T::Flt,
+                               ion_energy::Flt, nmax_explicit_sum::Integer,
```

Comment:
What's the reason for changing the argument order here?  It's not a problem, just curious.

Reply:
This was changed to reflect the argument order of the revised function signature of `ContinuumOpacity.H_I_bf(nH_I_div_partition, ν, ρ, T, ion_energy = _H_I_ion_energy, nmax_explicit_sum = 8, integrate_high_n = true)`. And I revised the signature of `ContinuumOpacity.H_I_bf` so that I could easily specify `nmax_explicit_sum` and `integrate_high_n` in a test.